src/test/ui/parser/raw-byte-string-literals.stderr

```diff
@@ -1,14 +1,20 @@
-error: raw byte string must be ASCII: \u{e9}
-  --> $DIR/raw-byte-string-literals.rs:2:8
+error: bare CR not allowed in string, use \r instead
```

Comment:
The "use \r instead" part is not applicable to raw strings.

Reply:
Good catch, fixed!